src/index.js

```diff
@@ -16,52 +16,42 @@ import { NotFoundPage } from './components/NotFound';
 import { LoginPage } from './components/Login';
 import { HomePage } from './components/Dashboard/';
 import { config } from './utils/oktaConfig';
+
 import LoadingComponent from './components/LoadingComponent';
+
 import UserProfile from '../src/components/Profile/UserProfile';
 import EditUserForm from '../src/components/Profile/EditUserForm';
+
 import CreateProgram from '../src/components/Program/CreateProgramForm';
 import EditProgram from '../src/components/Program/EditProgramForm';
+
 import CourseList from './components/Courses/CourseList';
 import AddCourseForm from './components/Courses/AddCourseForm';
 import EditCourseForm from './components/Courses/EditCourseForm';
+
 import ModuleList from './components/Modules/ModuleList';
 import ModuleText from './components/Modules/ModuleText';
 import AddModuleForm from './components/Modules/AddModuleForm';
 import EditModuleForm from './components/Modules/EditModuleForm';
 
 // redux
-import { createStore, applyMiddleware } from 'redux';
-import { persistStore, persistReducer } from 'redux-persist';
-import { PersistGate } from 'redux-persist/integration/react';
-import storage from 'redux-persist/lib/storage'; // defaults to localStorage for web
-import rootReducer from './state/reducers/rootReducer';
+// import { persistStore } from 'redux-persist';
+// import { PersistGate } from 'redux-persist/integration/react';
 import { Provider } from 'react-redux';
-import thunk from 'redux-thunk';
-import logger from 'redux-logger';
+
+import store from './state/store';
```

Comment:
moved all of the store logic to `src/state/store.js` b/c this `src/index.js` was looking rather busy